en/setup/dev_env_mac.md

```diff
@@ -31,14 +31,22 @@ After installing Homebrew, run these commands in your shell to install the commo
 ```sh
 brew tap PX4/px4
 brew install px4-dev
-# Optional, but recommended additional simulation tools:
+```
+To support simulation you you should also install
+```sh
 brew install px4-sim
 ```
-
-If the installation outputs an error message about missing requirements follow the instructions. Your system will be missing Java and Quartz:
-
 ```sh
-brew cask install xquartz java
+# Quartz
```

Comment:
@julianoes Do we still need quartz?

Reply:
That's a good question. Let me try without.

Reply:
I can't try right now but let's just assume it's not needed anymore. PX4 doesn't need it and the Gazebo docs don't mention it either.

Reply:
Hm, the docs here say it's still needed: https://raw.githubusercontent.com/Homebrew/install/master/install